tests/integration/blockchain/test_claim_commands.py

```diff
@@ -73,6 +73,12 @@ class ClaimSearchCommand(ClaimTestCase):
                 (result['txid'], result['claim_id'])
             )
 
+    async def assertFindsNoClaims(self, **kwargs):
+        kwargs.setdefault('order_by', ['height', '^name'])
+        results = await self.claim_search(**kwargs)
+        print(results)
+        self.assertEqual(0, len(results))
+
```

Comment:
Sorry - missed my print. 
I thought it was important that the logic for duration not return results if duration asked for was greater than that which existed.  I can remove it if you like.  I just added it because I wanted to make sure I wasn't getting false positives.

Reply:
You just do: `await self.assertFindsClaims([], ...)`